packages/angular/projects/ui-angular/src/lib/components/authenticator/components/sign-in/sign-in.component.html

```diff
@@ -38,14 +23,8 @@
 
 <amplify-slot name="sign-in-footer" [context]="context">
   <div data-amplify-footer>
-    <button
-      amplify-button
-      fontWeight="normal"
-      size="small"
-      variation="link"
-      fullWidth="true"
-      (click)="authenticator.toResetPassword()"
-    >
+    <button amplify-button fontWeight="normal" size="small" variation="link" fullWidth="true"
+      (click)="authenticator.toResetPassword()">
```

Comment:
Style nit -- why is this just two lines now? Shouldn't prettier break it up to multiple lines like it was before?

Reply:
Silly prettier, always changing its mind.